src/index.js

```diff
@@ -26,44 +25,101 @@ class Loader {
   stop () { this.$el.fadeOut('fast'); }
 }
 
+let pagesFlag = true;
+
 // ui elements
+const $html$body = $('html, body');
```

Comment:
It was the closest thing I could remember to describe the meaning of this multiple selector, because It's needed to apply on body or html depends on browser (firefox, chrome)